src/renderers/shared/fiber/ReactFiberContext.js

```diff
@@ -141,6 +143,25 @@ exports.pushTopLevelContextObject = function(fiber : Fiber, context : Object, di
 function processChildContext(fiber : Fiber, parentContext : Object, isReconciling : boolean): Object {
   const instance = fiber.stateNode;
   const childContextTypes = fiber.type.childContextTypes;
+
+  // TODO (bvaughn) Replace this behavior with an invariant() in the future.
+  // It has only been added in Fiber to match the (unintentional) behavior in Stack.
+  if (typeof instance.getChildContext !== 'function') {
+    if (__DEV__) {
+      const componentName = getComponentName(fiber);
+      
+      if (!warningAboutMissingGetChildContext[componentName]) {
+        warningAboutMissingGetChildContext[componentName] = true;
+        warning(
+          false,
+          'getChildContext() is not defined for %s',
```

Comment:
Maybe let’s make it a bit more descriptive? We can make it similar in style to messages in `ReactFiberClassComponent` (and Stack equivalents in `ReactCompositeComponent`). For example:

`'%s.childContextTypes is specified but there is no getChildContext() method on the instance. You can either define getChildContext() on %s or remove childContextTypes from it.',`